rezerwacje-front/src/views/Schedule/components/RoomsSelection/RoomsSelection.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/styles';
+import PropTypes from 'prop-types';
+import { 
+  Card, 
+  CardContent,
+  Grid, 
+  Typography, 
+  FormControl,
+  Select,
+  InputLabel,
+  Chip,
+  Input,
+  MenuItem,
+} from '@material-ui/core';
+
+const useStyles = makeStyles(theme => ({
+  formControl: {
+    width: '100%',
+  },
+  chips: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  chip: {
+    margin: 2,
+  },
+}));
+
+const RoomsSelection = props => {
+  const { className, ...rest } = props;
+
+  const classes = useStyles();
+
+  const rooms = props.rooms;
+
+  const [selectedRooms, setSelectedRooms] = React.useState(rooms.slice(0));
+
+  const handleSelectedRoomsChange = event => {
+    const value = event.target.value
+    setSelectedRooms(value)
```

Comment:
It seems that when you update rooms here the menu options change also, I guess they should not?